spec/workers/send_base_spec.rb

```diff
@@ -16,5 +16,15 @@
     expect(Workers::SendBase.new.send(:seconds_to_delay, 7)).to be >= 10_000
     expect(Workers::SendBase.new.send(:seconds_to_delay, 8)).to be >= 14_641
     expect(Workers::SendBase.new.send(:seconds_to_delay, 9)).to be >= 20_736
+    expect(Workers::SendBase.new.send(:seconds_to_delay, 10)).to be >= 28_561
```

Comment:
Is there any reason not to write this entire test as

```ruby
(2..19).each do |count|
  expect(Workers::SendBase.new.send(:seconds_to_delay, count)).to be >= ((count + 3)**4)
end
```

?

I really dislike this repeating source :/

Reply:
The reason was: I didn't want to test the formula with the same formula, so when there is an error in the formula at both places (because when you change it, you would just copy paste it to the test), we don't see it, while with explicit numbers, it is visible that the result is how we want it to be.

But I can change this, if you want. Maybe I can do it as you suggested, but then test 2, 9 and 19 with explicit numbers, so we still see that we have the expected increase. What do you think?

Reply:
Sounds good to me! Maybe a little comment why we test these cases "manually" wouldn't harm either. :)

Reply:
Updated, please review again :)